picard/ui/itemviews.py

```diff
@@ -339,8 +339,8 @@ def _add_other_versions():
                     for version in versions:
                         priority = {
                             "trackmatch": "0" if version['totaltracks'] == albumtracks else "?",
-                            "countrymatch": "0" if len(preferred_countries) == 0 or preferred_countries & set(version['countries']) else "?",
-                            "formatmatch": "0" if len(preferred_formats) == 0 or preferred_formats & set(version['formats']) else "?",
+                            "countrymatch": "0" if len(preferred_countries) == 0 or preferred_countries & set(version['countries'] or '') else "?",
```

Comment:
Do we really need `if len(preferred_countries) == 0` here (vs `if not preferred_countries`) ?